scripts/stakingSimulator.js

```diff
@@ -0,0 +1,580 @@
+const Helper = require("../test/helper.js");
+const BN = web3.utils.BN;
+const { DEPOSIT, DELEGATE, WITHDRAW, NO_ACTION } = require("./simulator/stakingActionsGenerator.js");
+const { expectRevert } = require('@openzeppelin/test-helpers');
+const StakeGenerator = require("./simulator/stakingActionsGenerator.js");
+
+//global variables
+//////////////////
+const { zeroBN, zeroAddress } = require("../test/helper.js");
+
+// for keeping score
+let depositRuns = 0;
+let delegateRuns = 0;
+let withdrawRuns = 0;
+let noActionRuns = 0;
+
+module.exports.doFuzzStakeTests = async function(
+    kyberStaking, NUM_RUNS, kncToken, stakers, epochPeriod
+) {
+    let result;
+    let validity;
+    for(let loop = 0; loop < NUM_RUNS; loop++) {
+        let operation = StakeGenerator.genNextOp(loop, NUM_RUNS);
```

Comment:
May be better to create 2 functions, one to execute command/operation, the other to verify data after.
Later when we create daoSimulator we can just use execute command/operation only, no need to verify?

Reply:
hmmm yes, will change